envs/sourmash.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+   - conda-forge
+   - bioconda
+   - defaults
+dependencies:
+   - sourmash-minimal=4.4.3
+   - pandas
```

Comment:
we should probably pin versions of all of these (like you did in `environment.yml`